### PR TITLE
🧪 [testing improvement] Add tests for getCommonMetrics

### PR DESCRIPTION
**Pull request by kurousa**
🎯 **What:** The testing gap addressed
- Added unit tests for the `getCommonMetrics` function in `formatters/DefaultFormatter.js`.

📊 **Coverage:** What scenarios are now tested
- **Complete activity:** Verified calculation of distance, time, elevation, and heart rate from a full activity object.
- **Missing fields:** Verified default values are used when fields like `distance` or `moving_time` are absent.
- **Heart rate logic:** Tested both scenarios where `has_heartrate` is true (returning the bpm) and false (returning "測定なし").
- **Zero values:** Ensured that zero values for distance, time, and elevation are handled correctly without falling back to defaults.

✨ **Result:** The improvement in test coverage
- Increased reliability of the core metrics calculation used across different activity formatters.
- Ensured that edge cases in activity data from the Strava API are gracefully handled.

---
*PR created automatically by Jules for task [15594229222999877100](https://jules.google.com/task/15594229222999877100) started by @kurousa*

### CODE REVIEW

File: tests/DefaultFormatter.spec.js [hunk 1]
```diff
@@ -13,9 +13,69 @@ global.CalendarApp = {
...
+        it('should handle missing optional fields with defaults', () => {
+            const activity = {};
+            const metrics = getCommonMetrics(activity);
+            expect(metrics).toEqual({
+                distanceKm: '0.0',
+                timeMin: 0,
+                elevation: 0,
+                hr: '測定なし'
+            });
+        });
```
[the remark this answers is not in the document]
Comment (kurousa) [reply tests/DefaultFormatter.spec.js]: > medium
> 
> activity オブジェクトの各プロパティ（distance, moving_time など）が null の場合のテストケースを追加することを検討してください。Strava API はデータが存在しない場合に null を返すことがあるため、空のオブジェクト（undefined）だけでなく null に対しても || 0 などのフォールバックが期待通りに動作することを確認しておくとより堅牢になります。
> 
> ```js
>         it('should handle missing or null optional fields with defaults', () => {
>             const activity = {
>                 distance: null,
>                 moving_time: null,
>                 total_elevation_gain: null,
>                 has_heartrate: null
>             };
>             const metrics = getCommonMetrics(activity);
>             expect(metrics).toEqual({
>                 distanceKm: '0.0',
>                 timeMin: 0,
>                 elevation: 0,
>                 hr: '測定なし'
>             });
>         });
> ```



> medium
> 
> activity オブジェクトの各プロパティ（distance, moving_time など）が null の場合のテストケースを追加することを検討してください。Strava API はデータが存在しない場合に null を返すことがあるため、空のオブジェクト（undefined）だけでなく null に対しても || 0 などのフォールバックが期待通りに動作することを確認しておくとより堅牢になります。
> 
> ```js
>         it('should handle missing or null optional fields with defaults', () => {
>             const activity = {
>                 distance: null,
>                 moving_time: null,
>                 total_elevation_gain: null,
>                 has_heartrate: null
>             };
>             const metrics = getCommonMetrics(activity);
>             expect(metrics).toEqual({
>                 distanceKm: '0.0',
>                 timeMin: 0,
>                 elevation: 0,
>                 hr: '測定なし'
>             });
>         });
> ```

@jules check this.
